Essentials/src/messages_cs.properties

```diff
@@ -512,6 +512,7 @@ teleportInvalidLocation=Souradnice nemohou prekrocit 30 000 000
 teleportNewPlayerError=§4Teleportace noveho hrace selhala\!
 teleportRequest=§c{0}§c se chce teleportovat k tobe.
 teleportRequestAllCancelled=§6Vsechny nevyrizene zadosti o teleport byli zruseny.
+teleportRequestCancelled=§6Your teleport request to §c{0}§6 was cancelled.
```

Comment:
Since the Crowdin integration, we now need to leave the translation files alone and only put new messages in `messages.properties`, else we'll get merge conflicts.